app/imports/components/posts/NewPost.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+import { addPost } from '/imports/api/photosMethods.js';
+
+export class NewPost extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      photo: this.props.photo,
+      caption: ''
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
```

Comment:
again if this just targets the text input I think it should be a bit more specific